src/palette/qml/MuseScore/Palette/internal/TreePaletteHeader.qml

```diff
@@ -147,78 +137,39 @@ Item {
         acceptedButtons: Qt.RightButton
 
         onClicked: {
-            toggleContextMenu()
+            menuLoader.toggleContextMenu(null)
```

Comment:
do we really need to call it if navigationParentControl is null? I'm not quite sure what would we expect in this case

Reply:
this is clicked on RightButton, does not participate in navigation 
I generally think this action is not necessary